### PR TITLE
INFRA: Initialize Unit Tests Project

### PR DESCRIPTION
**Pull request by ADILATIC**

### CODE REVIEW

File: Intern.Core.Api.Tests.Unit/DeleteMe.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+// ----------------------------------------------------------------------------------
+
+namespace Intern.Core.Api.Tests.Unit
+{
+    public class DeleteMe
+    {
+        [Fact]
+        public void ShouldBeTrue() => Assert.True(true);
+    }
+
```
Comment (glhays) [inline Intern.Core.Api.Tests.Unit/DeleteMe.cs:12]: @ADILATIC Remove extra line here.

File: Intern.Core.Api.Tests.Unit/Usings.cs [hunk 2]
```diff
@@ -0,0 +1 @@
+global using Xunit;
```
Comment (glhays) [inline Intern.Core.Api.Tests.Unit/Usings.cs:1]: @ADILATIC Remove this file. In the VS add project dialog there is a setting you need to uncheck, this file is automatically created. We do not use Global usings.

**Review by glhays — CHANGES_REQUESTED** [review]
@ADILATIC please see the review comment and correct. Good job this is much better.
You now have two Pull Requests open for the same task, @hassanhabib should he close the other one himself?

File: Intern.Core.Api.Tests.Unit/Usings.cs (hunk 2 above)
Comment (glhays) [reply Intern.Core.Api.Tests.Unit/Usings.cs:1]: @ADILATIC All you need to do now is delete it and commit the change on this branch.

**Comment by glhays** [comment]
@ADILATIC What issue does this close?
If you review the closed pull requests across Hassan's repositories you will see how it is done.
Sample:
Screenshot 2023-04-22 123338

**Comment by ADILATIC** [comment]
I didn't fully complete I was following the video . And that's where I created this pr.

**Comment by hassanhabib** [comment]
> @ADILATIC please see the review comment and correct. Good job this is much better. You now have two Pull Requests open for the same task, @hassanhabib should he close the other one himself?

I closed it @glhays

**Comment by glhays** [comment]
> I didn't fully complete I was following the video . And that's where I created this pr.

@ADILATIC I am not sure if this comment relates to my close comment but assuming it is.
I think you are confused some what, my comment relates to closing the ISSUE your Pull Request relates to. Your Issue is the Unit Test Initialization, So if you type in your comment box, Closes then a space, then a hash, you will get a popup listing the open issues. You can select or enter the issue number, When this pull request is merged it closes that issue by linking the PR to the Issue.

File: Intern.Core.Api.Tests.Unit/DeleteMe.cs (hunk 1 above)
Comment (hassanhabib) [inline Intern.Core.Api.Tests.Unit/DeleteMe.cs:12]: Remove extra line here

**Pull request merged**